sympy/sets/sets.py

```diff
@@ -428,7 +428,7 @@ def boundary(self):
         A point x is on the boundary of a set S if
 
         1.  x is in the closure of S.
-            I.e. Every neighborhood of x contains a point in S.
+            I.e. Every neighbourhood of x contains a point in S.
```

Comment:
I'd change it back. It's a shorter spelling and there's no reason to change the documentation from one accepted spelling to another when the first works perfectly fine.